app/scripts/views/sign_up.js

```diff
@@ -314,6 +315,7 @@ function (Cocktail, p, BaseView, FormView, Template, AuthErrors, mailcheck,
 
           if (preVerifyToken && account.get('verified')) {
             self.logViewEvent('preverified.success');
+            self.onSignInSuccess(account);
```

Comment:
Maybe [user.signInAccount](https://github.com/mozilla/fxa-content-server/blob/92425c5e0bf72779b9d51d447fcf47ddc2dad2df/app/scripts/models/user.js#L261) to trigger an event like `signin.success` with the account. The InterTabSignInMixin would liken for `signin.success`, check whether the account is verified, and then relay the message across the interTabChannel. 

I could imagine that working for logout as well.

Verification might be the trickiest case, much of the logic [in views/complete_sign_up](https://github.com/mozilla/fxa-content-server/pull/3206/files#diff-0dc8124c36f55037d013efdc72d8eaa8R68) (R68-R88) could move to user.js, and as part of that could trigger a `signin.success` message that is listened for by a mixin and relayed to the inter tab channel.